### PR TITLE
Fourth milestone complete

### PR DESCRIPTION
**Pull request by darshanjayadev**
In this milestone:

* Added [`Big.js`](https://www.npmjs.com/package/big-js) package.
* Implemented `operate.js` module.
* Implemented `calculate.js` module.
* Added `calculate.js` to `App` component.

Please take a look at below comments for more info.

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 import React from 'react';
+import calculate from '../../logic/calculate';
 import './App.css';
 import Display from '../Display/Display';
 import ButtonPanel from '../ButtonPanel/ButtonPanel';
 
 class App extends React.Component {
   render() {
+    const calculatorData = { total: null, next: null, operation: null };
+    const { total, next } = calculate(calculatorData, '0');
```
Comment (darshanjayadev) [inline src/components/App/App.js:11]: This is a temporary code, just to avoid lint error - `no-unused-vars`.

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    case 'AC':
+      total = null;
+      next = null;
+      operation = null;
+      modifiedCalculatorData = { total, next, operation };
+      break;
+    default:
+      next = next ? next + buttonName : buttonName;
+      modifiedCalculatorData = { total, next, operation };
+      break;
```
Comment (darshanjayadev) [inline src/logic/calculate.js:52]: The default case is for all the number buttons `[1, 2, 3, 4, 5, 6, 7, 8, 9, 0]`

File: src/logic/calculate.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    case '=':
+      total = total === null ? 0 : total;
+      next = next === null ? total : Number(next);
+      operation = operation === null ? '+' : operation;
+      total = operate(total, next, operation);
+      next = null;
+      operation = null;
+      modifiedCalculatorData = { total, next, operation };
+      break;
```
Comment (darshanjayadev) [inline src/logic/calculate.js:42]: Instead of testing the method, I imported it to Node REPL and tried it out ( to save on time ) and so far it has worked for most series of inputs. I think I'd be able to test it better in next milestone although I'm confident I've handled most edge cases.

image

File: src/logic/operate.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      total = Big(numberOne).plus(Big(numberTwo));
+      break;
+    case '-':
+      total = Big(numberOne).minus(Big(numberTwo));
+      break;
+    case 'X':
+      total = Big(numberOne).times(Big(numberTwo));
+      break;
+    case '÷':
+      total = Big(numberOne).div(Big(numberTwo));
```
Comment (shloch) [inline src/logic/operate.js:22]: dont forget to handle division by ZERO on next milestone when you use this module

**Review by shloch — APPROVED** [review]
Hello @juzQrios 

Good mastery of the project...Happy new year 🎅 🎅 
You can merge this, see you on next milestone 😊😊😊

File: src/logic/operate.js (hunk 4 above)
Comment (darshanjayadev) [reply src/logic/operate.js:22]: Sure
